src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/Resources.cs

```diff
@@ -62,7 +62,12 @@ public static byte[] GetResourceBytes(string name, string? folder = null)
             using var stream = GetResourceStream(name, folder);
 
             value = new byte[stream.Length];
+
+#if NET
+            stream.ReadExactly(value);
+#else
             stream.Read(value, 0, value.Length);
+#endif
```

Comment:
Did you consider a polyfill method `ReadExactly` to avoid this `#if NET`?

Reply:
Honestly, I wouldn't worry about adding a polyfill unless we need it in product code.